lib/sinon-match.js

```diff
@@ -54,7 +54,7 @@ function isMatcher(object) {
     return isPrototypeOf(matcher, object);
 }
 
-function matchObject(expectation, actual) {
+function matchObject(actual, expectation) {
```

Comment:
Why was this change made? Was it necessary?

Reply:
No, it wasn't strictly necessary, but I had to change the order of `actual` / `expected` in other places and I thought it should be consistent.